ocs_ci/utility/utils.py

```diff
@@ -2476,14 +2476,9 @@ def skipif_ocp_version(expressions):
         'True' if test needs to be skipped else 'False'
 
     """
-    skip_this = True
     ocp_version = get_running_ocp_version()
     expr_list = [expressions] if isinstance(expressions, str) else expressions
-    for expr in expr_list:
-        comparision_str = ocp_version + expr
-        skip_this = skip_this and eval(comparision_str)
-    # skip_this will be either True or False after eval
-    return skip_this
+    return any(ocp_version + expr for expr in expr_list)
```

Comment:
```suggestion
    return any((ocp_version + expr) for expr in expr_list)
```

Reply:
good catch :-) ... fixed in the new commit